spec.html

```diff
@@ -22894,15 +22928,14 @@ <h1>Static Semantics: ExportedBindings</h1>
         <emu-see-also-para op="ExportedBindings"></emu-see-also-para>
         <emu-grammar>
           ExportDeclaration :
-            `export` `*` FromClause `;`
-            `export` ExportClause FromClause `;`
+            `export` ExportFromClause FromClause `;`
         </emu-grammar>
         <emu-alg>
           1. Return a new empty List.
         </emu-alg>
-        <emu-grammar>ExportDeclaration : `export` ExportClause `;`</emu-grammar>
+        <emu-grammar>ExportDeclaration : `export` NamedExports `;`</emu-grammar>
         <emu-alg>
-          1. Return the ExportedBindings of |ExportClause|.
+          1. Return the ExportedBindings of |NamedExports|.
```

Comment:
This applies to the original text too but, you probably don't need this rule due to production chaining.

Reply:
would it be reasonable to leave it in for clarify?

Reply:
@ljharb and @zenparsing -- as editors, do you have an opinion about whether this should be left in or removed?

Reply:
For this PR I think you can leave it in. We might want to comb through and remove these chain productions in a future PR.